doc/nrf/releases_and_maturity/releases/release-notes-2.7.0.rst

```diff
@@ -0,0 +1,1238 @@
+
+* :ref:`bluetooth_mesh_light` sample:
+
+  * Added:
+
+    * Support for the :ref:`zephyr:nrf54l15pdk_nrf54l15` board.
+    * Support for DFU over Bluetooth Low Energy for the :ref:`zephyr:nrf54l15pdk_nrf54l15` board.
+    * Support for DFU over Bluetooth Low Energy for the :ref:`nRF5340 DK <ug_nrf5340>` board.
+
+* :ref:`ble_mesh_dfu_target` sample:
+
+  * Added:
+
+    * A note on how to compile the sample with new Composition Data.
+    * Point-to-point DFU support with overlay file :file:`overlay-ptp_dfu.conf`.
+    * Support for the :ref:`zephyr:nrf54l15pdk_nrf54l15` board.
+
+* :ref:`bluetooth_mesh_light_lc` sample:
+
+  * Added
+
+    * A section about the :ref:`occupancy mode <bluetooth_mesh_light_lc_occupancy_mode>`.
+    * Support for the :ref:`zephyr:nrf54l15pdk_nrf54l15` board.
+
+* :ref:`ble_mesh_dfu_distributor` sample:
+
+  * Added support for the :ref:`zephyr:nrf54l15pdk_nrf54l15` board.
+
+Cellular samples
+----------------
+
+* :ref:`at_client_sample` sample:
+
+  * Added support for Thingy:91 X.
+
+* :ref:`ciphersuites` sample:
+
+  * Updated the :file:`.pem` certificate for example.com.
+
+* :ref:`location_sample` sample:
+
+  * Removed ESP8266 Wi-Fi DTC and Kconfig overlay files.
+
+* :ref:`modem_shell_application` sample:
+
+  * Added:
+
+    * Support for sending location data details into nRF Cloud with ``--cloud_details`` command-line option in the ``location`` command.
+    * Support for Thingy:91 X Wi-Fi scanning.
+
+  * Removed ESP8266 Wi-Fi DTC and Kconfig overlay files.
+
+* :ref:`nrf_cloud_rest_cell_pos_sample` sample:
+
+  * Added the :ref:`CONFIG_REST_CELL_SEND_DEVICE_STATUS <CONFIG_REST_CELL_SEND_DEVICE_STATUS>` Kconfig option to control sending device status on initial connection.
+
+  * Removed the button press interface for enabling the device location card on the nRF Cloud website.
+    The card is now automatically displayed.
+
+* :ref:`modem_shell_application` sample:
+
+  * Removed sending GNSS UI service info to nRF Cloud.
+    This is no longer required by the cloud.
+
+* :ref:`nrf_cloud_multi_service` sample:
+
+  * Added:
+
+    * The ability to control the state of the test counter using the config section in the device shadow.
+    * Handling of L4 disconnect where CoAP connection is paused and the socket is kept open, then resumed when L4 reconnects.
+    * Checking in CoAP FOTA and shadow polling threads to improve recovery from communication failures.
+    * Sysbuild overlays for Wi-Fi and external-flash builds.
+
+  * Fixed:
+
+    * An issue that prevented network connectivity when using Wi-Fi scanning with the nRF91xx.
+    * An issue that caused the CoAP shadow polling thread to run too often if no data was received.
+
+* :ref:`udp` sample:
+
+  * Updated the sample to use the :c:macro:`SO_RAI` socket option with values :c:macro:`RAI_LAST` and :c:macro:`RAI_ONGOING` instead of the deprecated socket options :c:macro:`SO_RAI_LAST` and :c:macro:`SO_RAI_ONGOING`.
```

Comment:
```suggestion
  * Updated the sample to use the ``SO_RAI`` socket option with values ``RAI_LAST`` and ``RAI_ONGOING`` instead of the deprecated socket options ``SO_RAI_LAST`` and ``SO_RAI_ONGOING``.
```
Since these are not defined in the API docs.